torchquad/tests/trapezoid_test.py

```diff
@@ -86,6 +86,65 @@ def _run_trapezoid_tests(backend, _precision):
     for error in errors:
         assert error < 7000
 
+    if backend != "numpy":
```

Comment:
comment missing that the JIT section starts here (you had one in the others)

Reply:
done!